README.md

```diff
@@ -122,6 +122,22 @@ Dial by your location
 Meeting ID: 672 939 6170
 Find your local number: https://zoom.us/u/abrmtXWOh8
 
+#### Logs Sub-SIG
```

Comment:
perhaps "working group" instead of Sub-SIG

Reply:
```suggestion
#### Logs Working Group
```